File: go/1020.go
package main

import (
	"fmt"
)

func main() {
	var dias int
	fmt.Scan(&dias)
	ano := 365
	mes := 30

	anos := int(dias / ano)
	dias -= anos * ano
	meses := int(dias / mes)
	dias -= meses * mes

	fmt.Printf("%d ano(s)\n%d mes(es)\n%d dia(s)", anos, meses, dias)
}

File: Lua/Iniciante/1020.lua
-- Funcional no beecrowd
local A = io.read("n")
local anos = A // 365
A = A - anos * 365
local meses = A // 30
A = A - meses * 30
local dias = A

print(string.format("%d ano(s)", anos))
print(string.format("%d mes(es)", meses))
print(string.format("%d dia(s)", dias))